Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityProfileInspector.cs

```diff
@@ -43,25 +49,80 @@ protected virtual void OnEnable()
         /// <param name="property">the <see cref="Microsoft.MixedReality.Toolkit.BaseMixedRealityProfile"/> property.</param>
         /// <param name="guiContent">The GUIContent for the field.</param>
         /// <param name="showAddButton">Optional flag to hide the create button.</param>
+        /// <param name="serviceType"></param>Optional service type to limit available profile types.
         /// <returns>True, if the profile changed.</returns>
-        protected static bool RenderProfile(SerializedProperty property, GUIContent guiContent, bool showAddButton = true, Type serviceType = null)
+        public static bool RenderProfile(SerializedProperty property, GUIContent guiContent, bool showAddButton = true, Type serviceType = null)
         {
             return RenderProfileInternal(property, guiContent, showAddButton, serviceType);
         }
 
+        /// <summary>
+        /// Renders a non-ediable object field and an editable dropdown of a profile.
```

Comment:
>ediable [](start = 26, length = 7)

editable

Reply:
For little fixes like this will you do a code suggestion? Kurtis has been using them and they're amazing. You can merge fixes directly from the comment.

Reply:
Normally yes, for larger changes like this I've been using codeflow which doesn't have that integration.

---
In reply to: [279445503](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3773#discussion_r279445503) [](ancestors = 279445503)